dds_cli/account_manager.py

```diff
@@ -74,3 +79,62 @@ def add_user(self, email, role, project):
             )
 
         LOG.info(response_json.get("message", "User successfully added."))
+
+    def delete_user(self, email):
+        """Delete users from the system"""
+        # Perform request to API
+        json = {"email": email}
+
+        try:
+            response = requests.post(
+                dds_cli.DDSEndpoint.USER_DELETE,
+                headers=self.token,
+                json=json,
+            )
+
+            # Get response
+            response_json = response.json()
+            message = response_json["message"]
+
+        except requests.exceptions.RequestException as err:
+            raise dds_cli.exceptions.ApiRequestError(message=str(err))
+        except simplejson.JSONDecodeError as err:
+            raise dds_cli.exceptions.ApiResponseError(message=str(err))
+
+        # Format response message
+        if not response.ok:
+            if response.status_code == http.HTTPStatus.INTERNAL_SERVER_ERROR:
+                raise dds_cli.exceptions.ApiResponseError(message)
+            else:
+                raise dds_cli.exceptions.DDSCLIException(message)
+        else:
+            LOG.info(message)
+
+    def delete_own_account(self):
+        """Delete users from the system"""
+        # Perform request to API
+
+        try:
+            response = requests.post(
```

Comment:
Not a dead breaker but I think we should change `post` to `delete` here (and of course in the api)

Reply:
We can do this in another PR though

Reply:
Fixed. Will open the nano-PR to web soon!